frontend/src/features/admin-form/create/featureTour/utils.ts

```diff
@@ -0,0 +1,20 @@
+export const getFeaturTourToolTipMarginTop = (
+  featureTourStep: number,
+): string => {
+  let marginTop
+  switch (featureTourStep) {
+    case 0:
+      marginTop = '5rem'
+      break
+    case 1:
+      marginTop = '8.5rem'
+      break
+    case 2:
+      marginTop = '12rem'
+      break
+    default:
+      marginTop = '5rem'
+  }
```

Comment:
i suspect we do not need this weird margin prop. why is this needed? maybe discuss with designers too? seems like extra complexity for marginal gains.

Reply:
feels off, any reason why a custom tooltip will change the dynamic nature of the positioning? can you dig deeper into this?

Reply:
Solved it without using dynamic margins.